src/Worlds/DebugWorld/DebugWorld_DebugWorldActions.layer

```diff
@@ -71,4 +71,12 @@ GenericEntity {
    coords -0.124 0.902 -0.182
   }
  }
+}
+$grp GenericEntity : "{9B6B61BB3FE3DFB0}Prefabs/Items/Equipment/Radios/Radio_ANPRC77.et" {
+ Radio_ANPRC77_5CAB08C53F86CCBB {
+  coords 101.008 0.001 134.949
+ }
+ Radio_ANPRC77_5CAB08C53FE4BA26 {
+  coords 100.371 0.001 134.926
+ }
```

Comment:
```suggestion
```

Reply:
I would remove the radios again from debug world and if needed for testing rather add radios to the default loadout later on?

Reply:
Sure we can remove them again. Not sure if we should add radios to the default loadout, maybe if we can get a debug world only player loadout?